### ISSUE TITLE
video dock does not work to desktop 

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#21542]
seems like a regression, https://ampbyexample.com/advanced/advanced_video_docking/preview/ does not dock anymore.

**Comment by wassgha** [comment]
Can't reproduce, is this fixed?

**Comment by alanorozco** [comment]
Works for me. Will re-open on report.
